docs/logs/quick-start.md

```diff
@@ -0,0 +1,102 @@
+title: Sematext Logs Quick Start
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
+
+After you get logged into Sematext Cloud at <https://apps.sematext.com> (or <https://apps.eu.sematext.com> if using Sematext Cloud Europe), the first step is to create a Logs App. An App is an independent namespace for your data.
+
+For example, if you have a development and a production environment, it might make sense to have one App for each. You can create as many Apps as you want.
+
+## Creating a Logs App
+
+You create an App by pressing the **+ Create Logs App** button in the Logs tab.
+
+![Create a new Logs App](../images/guide/logs/sematext-logs-app-create.png)
+
+Add a name and press **Continue**.
+
+![Create a new Logs App Modal](../images/guide/logs/sematext-logs-app-create-modal.png)
+
+After creating a Logs App you will see an **integrations** screen that tells you how to [send data to your new Logs App](../logs/sending-log-events). 
+
+![Logs App Elasticsearch integration](../images/guide/logs/sematext-app-logs-elasticsearch.png)
+
+Once you start sending data, you can start [searching and analyzing these logs](../logs/searching-log-events) in the Logs App or [explore your data with Kibana](../logs/kibana).
+
+![Searching Log Events](../images/logs/logsene-ui.png)
+
+### Adding Data to Your Logs App
+
+There are two ways to send data to your Logs App: 
+
+- [Elasticsearch API](../logs/index-events-via-elasticsearch-api)
+- [Syslog](../logs/syslog)
+
+
+### Elasticsearch API
+
+The easiest way to **send logs is with** [Logagent](../logs/logagent), [Logstash](../logs/logstash), or Filebeat. Have in mind any log shipper will get the job done. You can also use **any tool that works with Elasticsearch's REST API**, for both [indexing](../logs/index-events-via-elasticsearch-api) and [searching](../logs/search-through-the-elasticsearch-api). 
+
+If you're using a particular **programming language**, configuring your **logging framework to send data to Sematext Logs** is also an option.
+
+The only condition is to **use the App's token as the index name**, and `https://logsene-receiver.sematext.com:443`, or `https://logsene-receiver.eu.sematext.com:443` as the Elasticsearch endpoint.
+
+Don't forget, if you're using **Docker**, setting up [Sematext Agent](../containers/sematext-agent/) is incredibly simple.
+
+Here's how to send a message from the terminal.
+```bash
+curl -XPOST https://logsene-receiver.sematext.com/YOUR-TOKEN-GOES-RIGHT-HERE/example/ -d '{
```

Comment:
@prog8 & @gr0 what is this /example/ bit?  Doc type?  Is that still required?

Reply:
@gr0 and @prog8 I'd love an answer here guys.

Reply:
Example is any type user decided to put there. After user specifies a type we convert it to `logsene_type` underneath (ES allows only single type and by forcing logsene_type we make sure there will be only one type in ES index). Fortunately we don't completely remove information about `example`. We store this information in ES doc in a field called `logsene_orig_type`.

Reply:
+1 to what @prog8 said. What we could point of is that Elasticsearch 7.0 allows only a single doc type, while Sematext Logs allows multiple types and we convert it internally allowing to filter on a type. For some people this may still be a thing to go for :)

Reply:
How do I say this in the simplest way possible?

> ...where `example` represents the value's type. It can be anything from Integer to String and anything in between. This value gets stored in the `logsene_type` field allowing for easy filtering on types when needed.

Feel free to give feedback here.

Reply:
Or...

> ...where `example` represents the desired type. It can be anything from `log`, `event`, `host`, `node`, and anything in between giving freedom to create custom types for logs. This value gets stored in the `logsene_type` field allowing for easy filtering on types when needed.

Reply:
PS. Synced with Pawel in Slack. We'll go with this :point_up: